packages/components/src/InputNumber/InputNumber.tsx

```diff
@@ -10,7 +10,7 @@ import { FormField, FormFieldProps } from "../FormField";
 interface InputNumberProps
   extends Pick<
     FormFieldProps,
-    Exclude<keyof FormFieldProps, "type" | "children" | "rows">
+    Exclude<keyof FormFieldProps, "type" | "children" | "rows" | "keyboard">
```

Comment:
Leaving this out of `InputNumber` will make `InputNumber` show the `text` mode. May want to consider using `decimal` in this case.

https://css-tricks.com/everything-you-ever-wanted-to-know-about-inputmode/#decimal

![image](https://user-images.githubusercontent.com/46460840/100918882-4ad2cd80-3496-11eb-9442-256c866b9e62.png)

Reply:
only including this attribute when keyboard prop is specified makes inputNumber a default numerical keyboard again based on `type`